docs/src/pages/guides/composition/ListRouter.tsx

```diff
@@ -17,9 +17,9 @@ import {
 import { Omit } from '@material-ui/types';
 
 interface ListItemLinkProps {
-  icon?: React.ReactElement;
-  primary: string;
   to: string;
+  primary: string;
```

Comment:
What's this change about?

Reply:
sorting properties ASC

Reply:
`to` > `primary`. This doesn't seem ascending to me but descending.